src/app/entity-groups/journal-entities/item-list-elements/sidebar-search-list-elements/journal/journal-sidebar-search-list-element.component.spec.ts

```diff
@@ -35,6 +38,11 @@ const parent = Object.assign(new Collection(), {
   },
 });
 
-describe('JournalSidebarSearchListElementComponent',
-  createSidebarSearchListElementTests(JournalSidebarSearchListElementComponent, object, parent, 'parent title', 'title', '1234, 5678'),
-);
+const expectedHierarchicalTitle = getExpectedHierarchicalTitle(parent, object);
+if (expectedHierarchicalTitle) {
+  expectedHierarchicalTitle.subscribe((hierarchicalTitle: string) => {
+    describe('JournalSidebarSearchListElementComponent', () => {
+      createSidebarSearchListElementTests(JournalSidebarSearchListElementComponent, object, parent, hierarchicalTitle, 'title', '1234, 5678');
```

Comment:
Since all of these tests call `createSidebarSearchListElementTests`, couldn't we move the call to `getExpectedHierarchicalTitle()` into that existing `createSidebarSearchListElementTests` method?  I don't see why we need to have the `if` or the `subscribe` above?  Is there something I'm missing here?